modules/pytket-qulacs/tests/test-requirements.txt

```diff
@@ -1 +1,3 @@
-git+git://github.com/quantumlib/OpenFermion@086073666d275de736861b41ed535a9fe75e63ac#egg=openfermion
+hypothesis==6.14.3
+pytest==6.2.4
```

Comment:
As these are automatically installed on the CI for all tests I don't think they need to be included here.

Reply:
I have removed these packages from pytket-qulacs' test-requirements.txt file